packages/cursorless-engine/src/actions/EditNew/runEditTargets.ts

```diff
@@ -59,44 +51,45 @@ export async function runEditTargets(
     .filter(({ range }) => range != null);
 
   const cursorIndices = cursorInfos.map(({ index }) => index);
+  const cursorRanges = cursorInfos.map(({ range }) => range!);
+  const editRanges = edits.map((edit) => edit.range);
 
-  const cursorSelections = {
-    selections: cursorInfos.map(({ range }) => range!.toSelection(false)),
-  };
-
-  const editSelections = {
-    selections: edits.map((edit) => edit.range.toSelection(false)),
-    rangeBehavior: RangeExpansionBehavior.openOpen,
-  };
-
-  const [
-    updatedThatSelections,
-    updatedCursorSelections,
-    updatedEditSelections,
-  ]: Selection[][] = await performEditsAndUpdateSelectionsWithBehavior(
+  const {
+    thatRanges: updatedThatRanges,
+    cursorRanges: updatedCursorRanges,
+    editRanges: updatedEditRanges,
+  } = await performEditsAndUpdateSelections({
     rangeUpdater,
     editor,
     edits,
-    [thatSelections, cursorSelections, editSelections],
-  );
+    preserveEditorSelections: true,
```

Comment:
I wonder if the word `cursor` is easier to understand, eg `preserveCursors` or `preserveCursorSelections`. I don't feel strongly, though; we're not super consistent. Just that `editorSelection` doesn't stronlgy evoke cursors to me

Reply:
renamed